### ISSUE TITLE
Deleting images outside of Damselfly - UI not updating

### ISSUE THREAD

**Issue by collowe** [Webreaper/Damselfly#349]
I deleted an image using a file manager (only image in dir).  In Damselfly I attempted to refresh the folder but it didn't pick up the removed image.  The image still remains, I can open in in Damselfly but it doesn't exist on disk.  At the moment, I can't see how I can remove images from Damselfly once catalogued?

**Comment by Webreaper** [comment]
Damselfly monitors the disk for changes and should re-index automatically when images are added or removed - it should disappear within a few minutes (you'll see 'indexing' in the statusbar). 

If that's not working, things to check:
- When you add new images, do they automatically appear?
- If you right-click the folder and choose 'Refresh' and 'Re-scan metadata' does the image disappear?
- If you restart Damselfly, does the image deletion get picked up by the initial index at startup?

Are there any errors in the log, particularly at startup, that indicate the FileWatchers aren't working correctly?

Can you also clarify the OS you're running on, and the Damselfly version.

**Comment by collowe** [comment]
Running on Ubuntu 20.04
Damselfly v2.9.4.0 (Web)
So, tested this - removed the only image in 2021/09/15 in file manager, went into Damselfly and right clicked > refresh > Re-scan metadata and I got a duplicate folder in the folder list (but not in the actual file structure).  The message said 1 new...0 removed.

I updated the version to your just release version (dockerised) and restarted and the duplicate folders were still there.  After a period the duplicate removed itself but left one 2021/09/15 behind.

If I add a new folder and add an image, e.g. 2021/09/16 it is not added to the folder list, nor is it when I refresh the page or leave it for a while

Tried to check the logs - the NAS this is hosting on went into a lot of disk activity,  Read lines counter went up then page refreshed and was just shown an empty page with a header. 

Damselfly.Web process currently using 1.1Gb of space but it doesn't seem stressed.
